frontend/src/pages/NewModel.tsx

```diff
@@ -17,6 +17,45 @@ import InteractiveTable from "../components/core/InteractiveTable";
 
 import { DateTime } from "luxon";
 
+function ProgressBar({
```

Comment:
Was holding off on making this a Core component since I know you're redesigning a few things, I just want something that works for the NewModel page we have rn

Reply:
Dope, maakes sense! This looks nice, need to hunt and gather to get colors for it eventually and redo the general color schema.